frontend/public/components/utils/tile-view-page.jsx

```diff
@@ -746,13 +746,20 @@ export class TileViewPage extends React.Component {
     const { filterGroupNameMap } = this.props;
     const { filterGroupsShowAll } = this.state;
 
+    const maxShown = 5;
+    const showMoreText = i18n.t('public~{{numRemaining}} more', {
+      numRemaining: Object.keys(filterGroup).length - maxShown,
+    });
+
     return (
       <FilterSidePanelCategory
         key={groupName}
         title={filterGroupNameMap[groupName] || groupName}
         onShowAllToggle={() => this.onShowAllToggle(groupName)}
         showAll={_.get(filterGroupsShowAll, groupName, false)}
         data-test-group-name={groupName}
+        maxShowCount={maxShown}
+        showText={showMoreText}
```

Comment:
This works great...
![Screen Shot 2022-06-01 at 11 31 40 AM](https://user-images.githubusercontent.com/8126518/171442906-5b9f9be5-7c0e-421d-abaf-609b299a213a.png)

However we also need to handle `hideText` prop as well...
![Screen Shot 2022-06-01 at 11 31 29 AM](https://user-images.githubusercontent.com/8126518/171442904-8da0c5f4-32e0-44e5-a3f0-5899cacae4b9.png)

This one doesn't need a variable, just the words `Show less` translated with `i18n.t`